# a sentence gets divided into phrases and for each
# phrase you get an intention and arguments, therefore
# in the case of having one phrase to recognize the lenght
# of this array should be 1
ActionSlot[] sentence_recognition
